### ISSUE TITLE
Accessibility: detach hover state from focused state

### ISSUE THREAD

**Issue by JunoNgx** [JunoNgx/justjot-frontend#31]
Currently, hovering cursor over an item changes it focused state. This is not correct and inaccessible (interferes with keyboard navigation; causes unpredictable focus; among other issues). This should not be the case, and the two should be separated.

**Comment by JunoNgx** [comment]
Resolved by #34
